Tests/OAuthenticatorTests/GoogleTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  GoogleTests.swift
+//  OceanLabsTests
+//
+//  Created by Martin Dufort on 2023-08-10.
+//
+
+import XCTest
+import OSLog
+@testable import OAuthenticator
```

Comment:
Is the `@testable` needed here?

Reply:
If I don't specify `@testable` than line 24 generates an error